src/connectors/clear-refinements/connectClearRefinements.ts

```diff
@@ -94,7 +94,10 @@ const connectClearRefinements: ClearRefinementsConnector = function connectClear
       transformItems = (items => items) as TransformItems<string>,
     } = widgetParams || {};
 
-    if (widgetParams.includedAttributes && widgetParams.excludedAttributes) {
+    if (
+      (widgetParams || {}).includedAttributes &&
+      (widgetParams || {}).excludedAttributes
+    ) {
```

Comment:
```suggestion
    if (
      widgetParams?.includedAttributes &&
      widgetParams?.excludedAttributes
    ) {
```

shorter? :)

Reply:
this is still doing four conditionals instead of my suggested 3

Reply:
You mean at the JavaScript engine level? Not sure how they evaluate that expression, but let's go with yours :)